diff --git a/node/router/src/heartbeat.rs b/node/router/src/heartbeat.rs
index a1ce8296a..fee3b7d73 100644
--- a/node/router/src/heartbeat.rs
+++ b/node/router/src/heartbeat.rs
@@ -36,15 +36,15 @@ pub trait Heartbeat<N: Network>: Outbound<N> {
     /// The duration in seconds to sleep in between heartbeat executions.
     const HEARTBEAT_IN_SECS: u64 = 25; // 25 seconds
     /// The minimum number of peers required to maintain connections with.
-    const MINIMUM_NUMBER_OF_PEERS: usize = 3;
+    const MINIMUM_NUMBER_OF_PEERS: usize = 4;
     /// The median number of peers to maintain connections with.
     const MEDIAN_NUMBER_OF_PEERS: usize = max(Self::MAXIMUM_NUMBER_OF_PEERS / 2, Self::MINIMUM_NUMBER_OF_PEERS);
     /// The maximum number of peers permitted to maintain connections with.
-    const MAXIMUM_NUMBER_OF_PEERS: usize = 21;
+    const MAXIMUM_NUMBER_OF_PEERS: usize = 100;
     /// The maximum number of provers to maintain connections with.
-    const MAXIMUM_NUMBER_OF_PROVERS: usize = Self::MAXIMUM_NUMBER_OF_PEERS / 4;
+    const MAXIMUM_NUMBER_OF_PROVERS: usize = Self::MAXIMUM_NUMBER_OF_PEERS / 10;
     /// The amount of time an IP address is prohibited from connecting.
-    const IP_BAN_TIME_IN_SECS: u64 = 300;
+    const IP_BAN_TIME_IN_SECS: u64 = 600;

     /// Handles the heartbeat request.
     fn heartbeat(&self) {

